docs/api-guide.md

```diff
@@ -706,14 +712,24 @@ Put put =
         .clusteringKey(clusteringKey)
         .floatValue("c4", 1.23F)
         .doubleValue("c5", 4.56)
-        .disableImplicitPreRead()
+        .enableImplicitPreRead()
         .build();
 ```
 
+Note that if you are certain that a record you are trying to mutate does not exist, you should not enable implicit pre-read for the `Put` operation for better performance. For example, if you load initial data, you should not enable implicit pre-read. A `Put` operation without implicit pre-read is faster than `Put` operation with implicit pre-read because the operation skips an unnecessary read.
```

Comment:
```suggestion
{% capture notice--info %}
**Note**

If you are certain that a record you are trying to mutate does not exist, you should not enable implicit pre-read for the `Put` operation for better performance. For example, if you load initial data, you should not enable implicit pre-read. A `Put` operation without implicit pre-read is faster than `Put` operation with implicit pre-read because the operation skips an unnecessary read.
{% endcapture %}

<div class="notice--info">{{ notice--info | markdownify }}</div>
```